demo/api/src/db/fixtures/fixtures.command.ts

```diff
@@ -105,6 +108,16 @@ export class FixturesCommand extends CommandRunner {
         const migrator = this.orm.getMigrator();
         await migrator.up();
 
+        await this.actionLogsService.runWithUserId(SYSTEM_USER_NAME, async () => {
```

Comment:
Do we need this method? Couldn't we call `this.actionLogsService.setUserId(SYSTEM_USER_NAME)` when initializing nest-commander?

Reply:
Does the action logs module work without a user id? Otherwise, we need to add this to every command and mention it in the docs.

Reply:
could it be added to api/src/console.ts?

Reply:
yes it can be added to console 👍 my thought about this was everyone has to think about it when it is needed then I've to add it. Not every entity for example has an log so therefore some commands dont need this context.